avocado/utils/ssh.py

```diff
@@ -1,3 +1,9 @@
+""" Provides utilities to carry out an SSH session.
+
+Example of use:
+
+    .. literalinclude:: /../../examples/apis/utils/ssh.py
```

Comment:
Besides the positive aspects of adding documentation, I have two comments here:

1) IMO, this is a bad example (courtesy of yours truly) for the module, because there's too much code not related to SSH itself
2) I'm not sure if using an include would help users looking at the documentation in any way but through readthedocs.org (think of `>>> help(avocado.utils.ssh)`).  Maybe a simpler example here is the way to go?

Reply:
I agree. going to change it on v2.

Reply:
@clebergnu as simple as that?

``` 
       from avocado.utils import ssh

        with ssh.Session(host, user='root', key='/path/to/file') as session:
            ret = session.cmd('ls')
            if ret.exit_status == 0:
                print(ret.decode())
```

Reply:
I think for the last line you intended something like:

```
print(ret.stdout_text)
```

Reply:
ok